client/my-sites/checkout/checkout/recent-renewals.jsx

```diff
@@ -3,26 +3,26 @@
  */
 import React from 'react';
 import { connect } from 'react-redux';
-import { localize } from 'i18n-calypso';
+import { useTranslate } from 'i18n-calypso';
 import PropTypes from 'prop-types';
-import moment from 'moment';
 
 /**
  * Internal dependencies
  */
 import QuerySitePurchases from 'components/data/query-site-purchases';
+import { useLocalizedMoment } from 'components/localized-moment';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { getSitePurchases } from 'state/purchases/selectors';
 
-function RecentRenewalListItem( { domain, productName, expiryMoment, translate } ) {
+function RecentRenewalListItem( { domain, productName, expiryString, translate } ) {
```

Comment:
Seems like `expiryString` isn't coming from anywhere right now. Did we forget to pass it down as a prop in `RecentRenewals`?

Reply:
Good catch! This was an incomplete rename.